iothub/service/src/IotHubServiceClient.cs

```diff
@@ -161,6 +161,13 @@ public IotHubServiceClient(string hostName, AzureSasCredential credential, IotHu
         /// <seealso href="https://docs.microsoft.com/azure/iot-develop/concepts-digital-twin"/>
         public DigitalTwinsClient DigitalTwins { get; protected set; }
 
+        /// <summary>
+        /// Subclient of <see cref="IotHubServiceClient"/> that handles getting, updating, and replacing device and module twins.
+        /// </summary>
+        /// <seealso href="https://docs.microsoft.com/azure/iot-hub/iot-hub-csharp-csharp-twin-getstarted"/>
+        /// <seealso href="https://docs.microsoft.com/azure/iot-hub/iot-hub-csharp-csharp-module-twin-getstarted"/>
+        public TwinsClient Twin { get; protected set; }
```

Comment:
I'm not sure where my previous comment went, but this should be plural like the others, right?

```suggestion
        public TwinsClient Twins { get; protected set; }
```